samples/rtio/ramp_throughput/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Please update to find_package(Zephyr)
```suggestion
find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
```